composer/callbacks/checkpoint_saver.py

```diff
@@ -240,7 +240,7 @@ def __init__(
         latest_remote_file_name: Optional[Union[str, pathlib.Path]] = '{run_name}/checkpoints/latest-rank{rank}.pt',
         save_interval: Union[Time, str, int, Callable[[State, Event], bool]] = '1ep',
         *,
-        overwrite: bool = False,
+        save_overwrite: bool = False,
```

Comment:
we can't rename these. That is an API breaking change.

Reply:
ah okay, i'll revert this.

Reply:
```suggestion
        overwrite: bool = False,
```